support-frontend/app/controllers/Application.scala

```diff
@@ -254,6 +258,28 @@ class Application(
       case _ => s"/uk/$path"
     }
   }
+
+  def checkout(countryGroupId: String): Action[AnyContent] = CachedAction() { implicit request =>
+    implicit val settings: AllSettings = settingsProvider.getAllSettings()
+
+    Ok(
+      views.html.main(
+        title = "Support the Guardian | Checkout",
+        mainElement = EmptyDiv("checkout"),
+        mainJsBundle = Left(RefPath("[countryGroupId]/checkout.js")),
+        mainStyleBundle = Right(StyleContent(Html(""))),
+      )(),
+    ).withSettingsSurrogateKey
```

Comment:
Make sure we put a `noindex` on this page.

Reply:
[Done](https://github.com/guardian/support-frontend/pull/5704/commits/abf63c7b07b94aa314f7d71994553b57b26b16b8)